tsstats/tests/test_ident_map.py

```diff
@@ -24,3 +25,19 @@ def test_ident_map(identmap_clients):
     assert clients['5'] == cl
     assert clients['UID1'] == uidcl
     assert clients['UID5'] == uidcl
+
+
+@pytest.fixture(scope="module")
+def pretty_identmap():
+    return [
+        {'primary_id': '1', 'alternate_ids': ['3', '6']},
+        {'primary_id': '4', 'alternate_ids': ['9', '42']}
+    ]
+
+
+def test_flip_pretty_ident_map(pretty_identmap):
```

Comment:
Please add tests for IdentMaps with additional, ignored keys (like `name`, as shown in the docs).
pytests [parametrizing of test functions](http://docs.pytest.org/en/latest/parametrize.html) should be suitable for that.